docs/guides/setup/configuring-data-docs/how-to-host-and-share-data-docs-on-azure-blob-storage.md

```diff
@@ -1,5 +1,103 @@
 ---
 title: How to host and share Data Docs on Azure Blob Storage
 ---
+import Prerequisites from '../../connecting_to_your_data/components/prerequisites.jsx'
 
-This article is a stub.
+This guide will explain how to host and share Data Docs on Azure Blob Storage. 
+Data Docs will be served using an Azure Blob Storage static website with restricted access.
+
+<Prerequisites>
+
+- [Set up a working deployment of Great Expectations](../../../tutorials/getting-started/intro.md)
+- Have permission to create and configured an Azure [storage account](https://docs.microsoft.com/en-us/azure/storage)
+
+</Prerequisites>
+    
+   
+<h3>Steps</h3>
+
+
+1. **Create an Azure Blob Storage static website.**
+    - Create a [storage account](https://docs.microsoft.com/en-us/azure/storage).
+    - In settings select [Static website](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-static-website-host) to display the configuration page for static websites.
+    - Select **Enabled** to enable static website hosting for the storage account.
+    - Write "index.html" in Index document.
+  
+  Note the Primary endpoint url. Your team will be able to consult your data doc on this url when you have finished this tuto. You could also map a custom domain to this endpoint.
+  A container called ``$web`` should have been created in your storage account.
+
+
+2. **Configure the** ``config_variables.yml`` **file with your azure storage credentials**
+
+  Get the [Connection string](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage?tabs=azure-portal) of the storage account you have just created.
+
+  We recommend that azure storage credentials be stored in the  ``config_variables.yml`` file, which is located in the ``uncommitted/`` folder by default, and is not part of source control. The following lines add azure storage credentials under the key ``AZURE_STORAGE_CONNECTION_STRING``. Additional options for configuring the ``config_variables.yml`` file or additional environment variables can be found [here](../../setup/configuring-data-contexts/how-to-configure-credentials-using-a-yaml-file-or-environment-variables.md).
+  
+  ```yaml
+ AZURE_STORAGE_CONNECTION_STRING: "DefaultEndpointsProtocol=https;EndpointSuffix=core.windows.net;AccountName=<YOUR-STORAGE-ACCOUNT-NAME>;AccountKey=<YOUR-STORAGE-ACCOUNT-KEY==>"
+  ```
+   
+
+3. **Add a new Azure site to the data_docs_sites section of your great_expectations.yml.**
+  
+    ```yaml
+  data_docs_sites:
+    local_site:
```

Comment:
The indentation did not render...

Reply:
Oh great catch, thank you! Should be updated now